resources/deploymentconfig-historical.yaml

```diff
@@ -51,6 +51,9 @@ spec:
         ports:
         - containerPort: 8080
           protocol: TCP
+        - name: metrics
+          containerPort: 2112
+          protocol: TCP
```

Comment:
I don't think we care about metrics on the historical instance (https://sippy-historical-bparees.apps.ci.l2s4.p1.openshiftapps.com/).  WDYT?

Reply:
Probably not, I'll remove.